### PR TITLE
Strip package used in module

### PR DESCRIPTION
**Pull request by SuperSandro2000**
This saves 4MB binary size.

### CODE REVIEW

File: flake.nix [hunk 1]
```diff
@@ -30,6 +30,7 @@
             version = "0.0.0";
             vendorHash = builtins.readFile ./tsnsrv.sri;
             src = with p; lib.sourceFilesBySuffices (lib.sources.cleanSource ./.) [".go" ".mod" ".sum"];
+            ldflags = [ "-s" "-w" ];
```
Comment (antifuchs) [inline flake.nix:33]: This is failing the alejandra check: https://github.com/boinkor-net/tsnsrv/actions/runs/7681585200/job/20934931225?pr=96#step:6:17 - could you run`nix fmt` and commit that?

Comment (SuperSandro2000) [reply flake.nix:33]: alejandra has in my opinion no sense of aesthetics but sure if you like it...

Comment (SuperSandro2000) [reply flake.nix:33]:
```suggestion
            ldflags = ["-s" "-w"];
```

**Comment by antifuchs** [comment]
Thanks for that, always appreciate a smaller binary (:

**Pull request merged**